### ISSUE TITLE
Clean and document all of the thermal comfort models in @stgeorges 's thermal indices component

### ISSUE THREAD

**Issue by chriswmackey** [ladybug-tools/ladybug-comfort#9]
As many ladybug users know, @stgeorges did a phenomenal job assembling [~18 different thermal comfort models](https://github.com/mostaphaRoudsari/ladybug/blob/master/src/Ladybug_Thermal%20Comfort%20Indices.py#L50-L69) in his [thermal comfort indices component](https://mostapharoudsari.gitbooks.io/ladybug-primer/content/text/components/Thermal_Comfort_Indices.html) for ladybug legacy.

At this point, this ladybug_comfort library has enough structure set up that the next step would be to transfer, clean and document these models to this library.  This will pave the way for a few things that are planned for the future:
1) Making [ladybug_comfort the go-to python library for thermal comfort models on PIP](https://pypi.org/project/ladybug-comfort/) and, though its documentation, making it a resource for people researching thermal comfort models.
2) Establishing workflows for people to easily take data in EPW format and analyze it using the comfort models, purely in Python.  An example of this using [an EPW and UTCI can be seen on the Readme page of this repo](https://github.com/ladybug-tools/ladybug-comfort/blob/development/README.md#usage) but having the other models will allow for possibilities like assessing Predicted Heat Strain from heat wave data, etc.
3) Setting a foundation to plot comfort polygons on charts like the psychrometric chart.  In legacy Ladybug, we only supported comfort polygons from the PMV model but I have already put some code in this library that will help plot polygons for UTCI.  I am thinking we can do this for a lot of the other models as well and the first step is to incorporate the models into this library.
4) Making it possible to build thermal comfort maps using the models in the future.  In legacy ladybug, we ran into challenges with this since most of the comfort models lived within one component but having them in a core ladybug library will make it possible for multiple components to reference these models, including comfort mapping components.

You can see that I have already done an example of translating one of the models from @stgeorges 's components with the [Heat Index (HI) in this library](https://github.com/ladybug-tools/ladybug-comfort/blob/master/ladybug_comfort/hi.py).  This was translated over from the [heat index function in the thermal indices component](https://github.com/mostaphaRoudsari/ladybug/blob/master/src/Ladybug_Thermal%20Comfort%20Indices.py#L572-L610) and the references in the docstring came from on [the wikipedia page for heat index](https://en.wikipedia.org/wiki/Heat_index).

I would be happy to translate the rest of the models but I am not be able to get to this until a few months down the line.  So I am posting this issue here in the event that someone sees it and decides that they would like to take up the mantle of transferring this code, documenting it, and getting it to conform to [PEP8](https://www.python.org/dev/peps/pep-0008/).  @stgeorges, you would undoubtedly be the most qualified to do this but I don't know what your schedule is like.  If anyone else has interest, please feel free to post on this issue.

**Comment by stgeorges** [comment]
Hi @chriswmackey,

Thank you for opening the issue, and providing it in a clear and explanatory way even with examples!
Indeed such code "gathering" may bring more functionalities and broaden the usage.
Sadly I not able to take on this task. My life had a twist in the last year and a half, both privately and professionally. This is how it is at the moment, I don't know how the things will develop in future.
The only tiny help (it it is help) that I can provide is just to mention the following: TCI component changes the names of its outputs and the descriptions of those outputs based on the chosen "_comfortIndex" input.
There is some useful information in those output descriptions. In the current component's code these output names and descriptions are located:
https://github.com/mostaphaRoudsari/ladybug/blob/master/src/Ladybug_Thermal%20Comfort%20Indices.py#L1668-L2155

Specifically the descriptions start on [line 1742](https://github.com/mostaphaRoudsari/ladybug/blob/master/src/Ladybug_Thermal%20Comfort%20Indices.py#L1742).

I apologize for not being able to contribute during these times.
I hope someone else might would be able to take the task.

**Comment by chriswmackey** [comment]
@stgeorges ,

I am sorry that I didn't respond to this months ago but hopefully it went with out saying that we completely  understand. You have already given so much to the community and you never have to apologize for not giving more. Rather, it is us who should all be thanking you. I hope the twist in your life doesn't hold you back from doing the things that you are passionate about and you should know that, if you ever want to get involved in what we'll be doing with your work here, just let us know.

**Comment by AntoineDao** [comment]
@chriswmackey do you reckon we can break up this issue into individual tasks and label them with `hacktoberfest`? I reckon this is a great task type to help people get involved in ladybug development and get to know the APIs a bit better. What do you think?

**Comment by chriswmackey** [comment]
That's a good idea @AntoineDao . I also realize that, for a couple of the models (notably PET), I would prefer to implement them or have someone with a good background in thermal comfort implement them.  So I can break out the ones that are particularly good for new contributors into separate issues and label them for hacktoberfest.

**Comment by stgeorges** [comment]
I am late for the reply on your comment @chriswmackey. Sorry for that.
I also thank you for the support, help and shared knowledge!
It is good know this! I will keep it in mind.

**Comment by chriswmackey** [comment]
I finished breaking this issue up into bite-size chunks with clear instructions and links to Djordje's code for each model. So these all should be suitable for new contributors to tackle and I gave all of the issues the `hacktoberfest` tag.  Given this, I think we can close this macro-level issue for now. 

And your response wasn't nearly as late as mine, @stgeorges , so definitely no worries.  Thank you again for all of your help and hopefully we can say at the end of this translation that this awesome library of comfort models you have assembled can go on to be used in even more applications, including many outside of Grasshopper!
